aggregator.js

```diff
@@ -10,30 +10,26 @@ class Aggregator {
     this.stats = {};
   }
 
+  _pushStats(path, value) {
+    if (!(value > 0)) {
```

Comment:
We should use 0 since else it would skewed right? Since we can do multiple runs now and you could get different result depending on what you get back, skipping 0 would skew it more than not skipping. Or how do you think? :)

Reply:
I'll fix this after, so we can push this in the coming fix release.